README.md

```diff
@@ -394,6 +389,18 @@ This project is intended to be a safe, welcoming space for collaboration, and
 contributors are expected to adhere to the
 [Contributor Covenant](http://contributor-covenant.org) code of conduct.
 
+### ART19 Gem Publishing
+
+Releases are manual, performed locally on a developer's machine. Gems are published to Github Packages. A comprehensive outline of this process can be found here: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.
+
+1. Increment the `ART19_REVISION` in [lib/jsonapi/version.rb#L5](https://github.com/art19/jsonapi.rb/blob/master/lib/jsonapi/version.rb#L4)
+
+2. Publish the gem:
```

Comment:
I think we may need to add to this -- 

I remember having to do 

bundle exec rake build

mv PACKAGE.VERSION  pkg/

bundle exec rake publish

unless this is not what worked for you when publishing and I am just doing it wrong

Reply:
bundle exec rake publish _should_ take care of everything -- I've been standardizing the same process across the jsonapi.rb, jsonapi-serializer, cequel and cassandra_driver gems. But, if something is off here we can correct the next time we publish.